js/geotrain.js

```diff
@@ -103,8 +115,28 @@ var map = new Array();
   canvas.addEventListener("click", ev_mouseclick, false);
   if (window.addEventListener){
     window.addEventListener("keydown", ev_keypress, false);
+    window.addEventListener("keyup", ev_keyrelease, false);
   } else if (window.attachEvent){ // IE sucks !
     window.attachEvent("keydown", ev_keypress, false);
+    window.attachEvent("keyup", ev_keyrelease, false);
+  }
+
+  tracklist = document.getElementById("geotracklist");
+
+  // type of event : mouseup, mousemove, mousedown, click,
+  tracklist.addEventListener("mousemove", cmd_mousemove, false);
+  tracklist.addEventListener("mouseover", cmd_hover, false);
+  // I want to follow start and end of selecting pieces, for a merge
```

Comment:
This comment apply to the "ev_mousedown" and "ev_mouseup" listener, you copied it and forget to delete or change it.